src/Package.appxmanifest

```diff
@@ -27,6 +27,9 @@
         <uap:SplashScreen Image="Assets\Preview\SplashScreen.png" />
       </uap:VisualElements>
       <Extensions>
+		<uap5:Extension Category="windows.startupTask" Executable="$targetnametoken$.exe" EntryPoint="$targetentrypoint$">
+		  <uap5:StartupTask TaskId="DevHomeDefaultStartup" Enabled="false" />
+		</uap5:Extension>

```

Comment:
nit, indentation

doesn't this require an app execution alias to make sure the thing that gets launched is the package not the exe?

Reply:
I guess I'm not sure how the PR would work as-is, does it actually start the app? if so, do we understand how that happens?

Reply:
https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.startuptask?view=winrt-22621 does not show an AEA.

Reply:
hm that's curious. @jonwis is the sample correct?

Reply:
For Win32 startup tasks, you need the `EntryPoint` attribute to be `Windows.FullTrustApplication` as the docs show - that makes the system use CreateProcess activation instead of trying the UWP CoreApplication launch path.  The `Executable` needs to be the name of an exe in your package.  I recently tried this elsewhere and found that `$targetnametoken$` didn't get replaced here automatically, so I had to manually (boo!) put my exe name in.

Reply:
great. and the system realizes that the executable needs to be launched in the context of the package, not as an unpackaged exe, right?

Reply:
Yep, and just FYI that startup state gets scribbled into `HKCU\Software\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppModel\SystemAppData\[pfn]\[taskid]\*`